src/index.js

```diff
@@ -365,13 +494,11 @@ function figureMove (idIn) {
 
         if (firstColor !== secondColor && boardSquare[squareId].canMove) {
           console.log('by enemy');
-          figureRemove(squareId);
+          // figureRemove(squareId);
           figurePositionChange(squareId, idIn);
         } else { console.log('by ally'); }
       }
-    } catch (error) {
-
-    }
+    } catch (error) { console.log('ERROR in figureMove()'); }
```

Comment:
kinda useless catch

Reply:
Same here, sometimes boardSquare[SOMETHING] will not have and element in it, and trying accessing any property will lead to and error